topics/scientific-computing/index.md

```diff
@@ -0,0 +1,7 @@
+---
+display_name: Scientific computing
+short_description: Scientific computing is the use of computers to solve scientific problems.
+topic: scientific-computing
+wikipedia_url: https://en.wikipedia.org/wiki/Scientific_computing
+---
+Scientific computing is the use of computers to solve scientific problems. It is a subfield of computational science and computational engineering. It is also a subfield of computer science, in particular computer science that is concerned with the application of algorithms to scientific problems.
```

Comment:
I know this is a vague topic but I think this description is still a bit too vague... it doesn't actually say anything. Maybe pull in some verbiage from the wikipedia article you've linked?

> It involves the development of models and simulations to understand natural systems.

Reply:
Mearman:topic/new/scientific-computing